components/docs-chef-io/content/automate/servicenow_reference.md

```diff
@@ -376,3 +377,74 @@ The ServiceNow app deletes these records of the corresponding closed incidents,
 `x_chef_incident.logging.verbosity`
 
 : Used to debug the data in ServiceNow. Enables the selected logging level and is visible in logs. The possible values are `debug`, `warn`, `info`, and `error`. Default: `error`.
+
+## Event Creation App Roles
+
+You can associate a single user with more than one role.
+
+### Role `x_chef_event.admin`
+
+You can assign the `x_chef_event.admin` role to a user other than a System Administrator to allow another user to manage the application properties and logs.
+
+The System Administrator authorization includes access to the tasks in the **Admin** role.
+
+The **Admin** role grants user access to the following:
+
+- Chef Events menu
+- Client Infra Client Run Alerts menu item
+- Chef InSpec scans Alerts menu item
+- Client Infra Client Runs
+- Chef InSpec scans
+- Properties menu item
+- Support menu item
+- Logs menu item
+
+### Role `x_chef_event.chef_inspec_scans_user`
+
+The `x_chef_event.chef_inspec_scans_user` role is suitable for those users who require application access without administration rights. The role grants a user access to the:
+
+- Chef Events menu
+- Chef Infra scans Alerts menu item
+- Chef InSpec scans menu item
+- Support menu item
+
+{{< note >}}
+
+Client run and Chef InSpec scan records are linked to events, and is appropriate to assign this role to users who manage events in ServiceNow.
+
+{{< /note >}}
+
+### Role x_chef_event.api
+
+The `x_chef_event.api` role is suitable for users responsible for integrating the Chef Automate data into ServiceNow. We recommend creating a new user specifically for this role. The Chef Automate Event Creation App requires the API role to set up communication with Chef Automate.
+
+### Event Creation App Properties
+
+`x_chef_event.client_run_message_key`
+: Used to associate a Chef Infra client run record with an event record. Possible values: `cookbook` and `node`. Default: `cookbook`.
+
+   Setting the value to `cookbook` creates an event for the failed cookbook. This associate all failing Chef Infra client runs with the corresponding event. `cookbook` is the default value because the number of nodes exceeds the number of cookbooks in any system. The message key description of the event provides information about the failure.
```

Comment:
Simplify. Also, please don't indent the first line of paragraphs. In Markdown, they should start flush on the left. If you need to move the paragraph in one level, then VSCode should move the entire thing with the tab key.

```suggestion
Set the value to `cookbook` to create an event for a failed cookbook. This associates all failing Chef Infra client runs with this cookbook failure with the event. `cookbook` is the default value because the number of nodes exceeds the number of cookbooks in any system. The message key description of the event provides information about the failure.
```

The statement "the number of nodes exceeds the number of cookbooks in any system" isn't necessarily true. It might be better to say that the number of cookbooks is independent of the number of nodes in any system. We might be able to drop the sentence entirely, it is the default value simply because that's what it is.

I'm sure we use it in the other documents, but we should use the pattern:

- `Set the value to X to create Y` ("Set the value to `cookbook` to create an event for a failed cookbook") 

instead of:
- `setting the value to X creates Y` ("Setting the value to `cookbook` creates an event for the failed cookbook")